apps/client/src/shared/component/Header/Notification/NotificationList.tsx

```diff
@@ -0,0 +1,64 @@
+import { useNotificationsQueryObject } from "@/app/api/notifications/query";
+import { IcnBtnArrowDown } from "@/asset/svg";
+import Empty from "@/shared/component/Empty";
+import type { NotificationType } from "@/shared/component/Header/Notification";
+import NotificationListItem from "@/shared/component/Header/Notification/NotificationItem";
+import {
+  emptyWrapper,
+  moreButtonStyle,
+  ulStyle,
+} from "@/shared/component/Header/Notification/index.css";
+import { useSuspenseQuery } from "@tanstack/react-query";
+
+interface NotificationListProps {
+  notificationType: NotificationType;
+  isExpanded: boolean;
+  expandList: () => void;
+}
+
+const NotificationList = ({
+  notificationType,
+  isExpanded,
+  expandList: handleExpandList,
+}: NotificationListProps) => {
+  const { data: notificationData } = useSuspenseQuery(
+    useNotificationsQueryObject(notificationType),
+  );
+
+  const notificationList = isExpanded
+    ? notificationData
+    : notificationData?.slice(0, 6);
+
+  const shouldShowMoreButton = notificationList.length >= 6 && !isExpanded;
+
+  return notificationList.length > 0 ? (
+    <>
+      <ul className={ulStyle} aria-label="알림 목록">
+        {notificationList.map((notification) => (
+          <NotificationListItem
+            key={notification.id}
+            {...notification}
+            notificationType={notificationType}
+          />
+        ))}
+      </ul>
+
+      {shouldShowMoreButton && (
+        <button
+          className={moreButtonStyle}
+          onClick={handleExpandList}
+          aria-expanded={isExpanded}
+        >
+          <IcnBtnArrowDown width={"1.2rem"} height={"1.2rem"} />
```

Comment:
```suggestion
          <IcnBtnArrowDown width={"1.2rem"} height={"1.2rem"} role="presentation"/>
```

장식 요소에는 presentation role 지정해주면 좋을 것 같아요!

Reply:
aira-hidden="true"로 설정해주었어요!